### DISCUSSION TITLE
Langchain error ERR_Network

### DISCUSSION THREAD

**Discussion by KelvinHYYip** [logancyang/obsidian-copilot#148] [General]
Hi Longan, 

I started using copilot recently and was testing its functionality today.
It worked before but not today.
I received a pop up banner: "Langchain error ERR_Network". May I ask what does it mean?

Note: the same error occurred before and after the recent Obsidian update V1.4.13.

Regards,
Kelvin

**Comment by logancyang** [comment]
I sometimes get this when the network connection is poor, or simply because OpenAI API is down.
